code/modules/power/generators/turbine.dm

```diff
@@ -44,7 +47,18 @@
 	var/capacity = 1e6
 	var/comp_id = 0
 	var/efficiency
-
+	///value that dertermines the amount of overheat "damage" on the turbine.
+	var/overheat = 0
+	///Required overheat "damage" to proc an overheat event. By default, this is seconds of overdrive required for an overheat event.
+	var/overheat_threshold = 300
+	///This value needs to be zero. It represents seconds since the last overheat event
+	var/last_overheat = 0
+	//Internal radio, used to alert engineers of turbine trip!
+	var/obj/item/radio/radio
+	///overheat message
+	var/overheat_message = "Alert! The gas turbine generator's bearings have overheated. Initiating automatic cooling procedures. Manual restart is required."
+	///expresses time until next bearing overheat as a percentile
```

Comment:
```suggestion
	/// value that dertermines the amount of overheat "damage" on the turbine.
	var/overheat = 0
	/// Required overheat "damage" to proc an overheat event. By default, this is seconds of overdrive required for an overheat event.
	var/overheat_threshold = 300
	/// This value needs to be zero. It represents seconds since the last overheat event
	var/last_overheat = 0
	// Internal radio, used to alert engineers of turbine trip!
	var/obj/item/radio/radio
	/// overheat message
	var/overheat_message = "Alert! The gas turbine generator's bearings have overheated. Initiating automatic cooling procedures. Manual restart is required."
	/// expresses time until next bearing overheat as a percentile
```

Reply:
resolved in 2a07ab7